scripts/detect_bad_deps.js

```diff
@@ -41,7 +41,7 @@ exec("flow check --profile", function(error, stdout, stderr) {
     process.exit(1);
   }
   console.log("Biggest cycle: " + cycle_len);
-  let MAX_CYCLE_LEN = 56;
+  let MAX_CYCLE_LEN = 57;
```

Comment:
Flow reported the new added file ECMAScriptFunctionValue.js as new cycle. Considering both BoundFunctionValue and NativeFunctionValue are also in the list, I thought it is expected ECMAScriptFunctionValue.js identified as new cycle? Do you mind share some alternative choices/solutions to avoid the increase of cycle?